### ISSUE TITLE
False positives on Duplicate Form Label

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#950]
@SteveJonesDev @pattonwebz, doi you expect this to flag duplicate form labels?

`<input type="text" aria-labelledby="myBillingId myNameId">`

https://a11ycheckrules.wpenginepowered.com/broken-aria-reference/?edac=1196&edac_nonce=c716af526f

This came up while testing the alpha. I'm not certain its correct to flag this as a duplicate label.

**Comment by pattonwebz** [comment]
This should fail because there are 2 labels here according to the way the rule is written, we can refactor to actually check if the id exists and for it's text content.
